tests/integration/targets/test_manage_transit_gateway/defaults/main.yml

```
@@ -0,0 +1,15 @@
+_resource_prefix: 'AnsibleTest'
+cidr_prefix: '10.{{ 255 | random(seed=_resource_prefix) }}'
+tgw_name: '{{ _resource_prefix }}'
+vpc_name_a: '{{ _resource_prefix }}-1'
+vpc_name_b: '{{ _resource_prefix }}-2'
+vpc_cidr_a: '{{ cidr_prefix }}.1.0/24'
+vpc_cidr_b: '{{ cidr_prefix }}.2.0/24'
+
+subnet_cidr_a_1: '{{ cidr_prefix }}.1.0/26'
+subnet_cidr_b_1: '{{ cidr_prefix }}.2.0/26'
+
+subnet_name_a_1: '{{ _resource_prefix }}-a-1'
+subnet_name_b_1: '{{ _resource_prefix }}-b-1'
+
+attachment_name_complex: '{{ _resource_prefix }}-complex'
```

```suggestion
cidr_prefix: '10.{{ 255 | random(seed=resource_prefix) }}'
tgw_name: '{{ resource_prefix }}'
vpc_name_a: '{{ resource_prefix }}-1'
vpc_name_b: '{{ resource_prefix }}-2'
vpc_cidr_a: '{{ cidr_prefix }}.1.0/24'
vpc_cidr_b: '{{ cidr_prefix }}.2.0/24'

subnet_cidr_a_1: '{{ cidr_prefix }}.1.0/26'
subnet_cidr_b_1: '{{ cidr_prefix }}.2.0/26'

subnet_name_a_1: '{{ resource_prefix }}-a-1'
subnet_name_b_1: '{{ resource_prefix }}-b-1'

attachment_name_complex: '{{ resource_prefix }}-complex'
```

The `resource_prefix` variable is created by ansible-test for every `cloud/aws` test type, using a hardcoded version may create collision if multiple instances of the targets are running at the same time